src/quantum/plugin/QuantumPlugin.ts

```diff
@@ -24,6 +24,11 @@ export class QuantumPluginClass implements Plugin {
                     // remove uglify js
                     delete plugins[index];
                 }
+                if (plugin.constructor.name === "UglifyESPluginClass") {
+                    this.coreOpts.uglify = { es6:true, ...plugin.options };
```

Comment:
As far as I remember UglifyEs5 will complain about an unknown option.  When passing the option, let's make sure `es6 :true` is not there (delete it)

It would great if you added documentation for that here:
https://github.com/fuse-box/fuse-box/blob/master/docs/quantum.md#installation

Good job!

Reply:
It got extracted later on via destructuring here: `{ es6 = false, ...opts }`

Reply:
Ok, can you add the docs please?